extensions/cassandraql/runtime/pom.xml

```diff
@@ -51,6 +51,21 @@
         <dependency>
             <groupId>com.datastax.oss.quarkus</groupId>
             <artifactId>cassandra-quarkus-client</artifactId>
+            <exclusions>
```

Comment:
This slipped into by mistake. It was some autogen code. Not sure if it can stay in this PR or we better remove. @jamesnetherton

Reply:
Nevermind, I've rebased and it fixed the code.